style.css

```
@@ -0,0 +1,100 @@
+body {
+  background-color: rgb(253, 251, 251);
+  position: absolute;
+  top: 170px;
+}
+
+header {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  text-align: center;
+  background-color: gray;
+}
```

Olá, @biazidioti! O que você acha de trazer o cabeçalho para a frente de todo o conteúdo, para que nenhum outro o sobreponha? Eu acho que ficaria ainda mais daora! Vê aí o que tu acha! :rocket: 

```suggestion
header {
  position: fixed;
  top: 0;
  width: 100%;
  text-align: center;
  background-color: gray;
  z-index: 1;
}
```

---

> Olá, @biazidioti! O que você acha de trazer o cabeçalho para a frente de todo o conteúdo, para que nenhum outro o sobreponha? Eu acho que ficaria ainda mais daora! Vê aí o que tu acha!

Òtima ideia!!! Vou providenciar essa mudança. 
Muito obrigada, Wanderson.